rollup.config.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-env node */
+import typescript from "rollup-plugin-typescript2";
+import multiInput from "rollup-plugin-multi-input";
+import fs from "fs";
+import path from "path";
+
+function CJS() {
+  return {
+    input: [
+      "src/Button",
```

Comment:
It seems strange that we'd need to modify our build config every time we add something to `src` - can we just grab everything in there?

Reply:
You're right. I made this change. It makes the config a little more confusing, but will be future-proof. Thanks!

Reply:
Answering your question more directly, we can't grab everything. We have to exclude certain things, but that's fine! We can exclude scripts/ directories, stories, and test files easily.